controls/SV-257949.rb

```diff
@@ -40,7 +40,7 @@
       end
     end
   else
```

Comment:
<img width="2017" alt="image" src="https://github.com/user-attachments/assets/a60f1705-e058-4c76-b2f9-94c2b8978795">

seems like it's actually bugging out because it's looking for a file not a string.

Reply:
Doing some more research, it's expecting that if something is passed in that's not a hash, then it's a file path.

the ini one basically is just building off of the json one: https://github.com/inspec/inspec/blob/main/lib/inspec/resources/ini.rb#L5

the constructor for the json one calls out to a wrapper parse function: https://github.com/inspec/inspec/blob/main/lib/inspec/resources/json.rb#L37

the check to see if it's a hash otherwise it treats it like a path which is the codepath we're currently following: https://github.com/inspec/inspec/blob/main/lib/inspec/resources/json.rb#L37

the codepath we actually want where we treat the input as just a raw file: https://github.com/inspec/inspec/blob/main/lib/inspec/resources/json.rb#L89